packages/core/test/Form.test.jsx

```diff
@@ -2,8 +2,9 @@ import * as React from 'react';
 import { expect } from 'chai';
 import sinon from 'sinon';
 import { createRef } from 'react';
-import { renderIntoDocument, act, Simulate } from 'react-dom/test-utils';
-import { render, findDOMNode } from 'react-dom';
+import { fireEvent, act, render } from '@testing-library/react';
+import { Simulate } from 'react-dom/test-utils';
```

Comment:
Simulate is still being used?

Reply:
Thanks for taking the time to review @heath-freenome.  There are two cases where _fireEvent_ did not behave as I had expected and that's why I continued to use Simulate.  

- In the case of a **select** component with the option to select multiple values. I simply couldn't figure out how to set the  **selected** attribute on more than one option element.  

- In the case of an input field of type number. fireEvent will not set a value to a non numeric.  

I'm happy to take any suggestions.

Reply:
Just wanted to follow up on the React upgrade. I've been developing quite a bit on top of the React 18 library and haven't had any major issues. There are peer dependency errors that I've come across in libraries that are fixed to React 17.   You'll also see the following warning:
```Warning: ReactDOM.render is no longer supported in React 18.```